build.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+set -eu
+
+is_substring()
+{
+    case "$2" in
+        *$1*)
+        return 0
+        ;;
+        *)
+        return 1
+        ;;
+    esac
+}
+
+OK=0
+CURRENT_DIR=$(pwd)
+MY_NAME=$(basename -- "$0")
+ROOTFS_ARCHIVE="rootfs.tar.xz"
+BUILD_DIRECTORY="${CURRENT_DIR}/build"
+ROOTFS_DIRECTORY="${BUILD_DIRECTORY}/rootfs"
+DEBIAN_VERSION="jessie"
+
+cleanup()
+{
+    if is_substring "${CURRENT_DIR}" "${BUILD_DIRECTORY}"
+    then
+        rm -rf "${BUILD_DIRECTORY}"
+    fi
+}
+
+prepare()
+{
+    if [ ! -d "${ROOTFS_DIRECTORY}/usr/bin" ]
+    then
+        mkdir -p "${ROOTFS_DIRECTORY}/usr/bin"
+    fi
+
+    if [ ! -f "${ROOTFS_DIRECTORY}/usr/bin/qemu-arm-static" ]
+    then
+        cp /usr/bin/qemu-arm-static "${ROOTFS_DIRECTORY}/usr/bin/"
```

Comment:
also, your usage of command -v "${QEMU_STATIC_BIN}" in the other script below is far more sexy!